python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -1,278 +1,463 @@
+cloud.setLegend("Observations")
+cloud.setPointStyle(
+    ot.ResourceMap.GetAsString("CalibrationResult-ObservationPointStyle")
+)
+graph.add(cloud)
+graph.setColors(ot.Drawable.BuildDefaultPalette(2))
+graph.setLegendPosition("topleft")
+view = viewer.View(graph)
 
 # %%
-# The `LinearLeastSquaresCalibration` class performs the linear least squares calibration by linearizing the model in the neighbourhood of the reference point.
+# We see that the observations are relatively noisy, but that the
+# trend is clear: this shows that it may be possible to fit the model.
+# At this point, we have a data set that we can use for calibration
+# and a model to calibrate.
+
 
 # %%
+# Calibration with linear least squares
+# -------------------------------------
+# The :class:`~openturns.LinearLeastSquaresCalibration` class performs the
+# linear least squares
+# calibration by linearizing the model in the neighbourhood of the reference point.
 algo = ot.LinearLeastSquaresCalibration(
     mycf, observedStrain, observedStress, thetaPrior, "SVD"
 )
 
 # %%
-# The `run` method computes the solution of the problem.
-
-# %%
+# The :meth:`~openturns.LinearLeastSquaresCalibration.run` method computes
+# the solution of the problem.
 algo.run()
-
-# %%
 calibrationResult = algo.getResult()
 
 # %%
 # Analysis of the results
 # -----------------------
-
-# %%
-# The `getParameterMAP` method returns the maximum of the posterior distribution of :math:`\theta`.
-
-# %%
+# The :meth:`~openturns.CalibrationResult.getParameterMAP` method
+# returns the maximum of the posterior distribution of :math:`\theta`.
 thetaMAP = calibrationResult.getParameterMAP()
-thetaMAP
+print("theta After = ")
+print("    R = %.2f (MPa)" % (thetaMAP[0] / 1.0e6))
+print("    C = %.2f (MPa)" % (thetaMAP[1] / 1.0e6))
+print("    Gamma = %.4f" % (thetaMAP[2]))
+print("theta Before = ")
+print("    R = %.2f (MPa)" % (thetaPrior[0] / 1.0e6))
+print("    C = %.2f (MPa)" % (thetaPrior[1] / 1.0e6))
+print("    Gamma = %.4f" % (thetaPrior[2]))
+print("theta True = ")
+print("    R = %.2f (MPa)" % (cm.trueR / 1.0e6))
+print("    C = %.2f (MPa)" % (cm.trueC / 1.0e6))
+print("    Gamma = %.4f" % (cm.trueGamma))
 
 # %%
 # We can compute a 95% confidence interval of the parameter :math:`\theta^\star`.
-
-# %%
 thetaPosterior = calibrationResult.getParameterPosterior()
 thetaPosterior.computeBilateralConfidenceIntervalWithMarginalProbability(0.95)[0]
 
 # %%
-# We can see that the `Gamma` parameter has a large confidence interval : even the sign of the parameter is unknown.
+# We can see that the `Gamma` parameter has a large confidence interval:
+# even the sign of the parameter is unknown.
 # The parameter which is calibrated with the smallest confidence
-# interval is `R`, which confidence interval is [708.3,780.0] (MPa).
-# This is why this parameter seems the most important in this case.
+# interval is `R`, which confidence interval is [741, 758] (MPa).
```

Comment:
Does this make sense? R, C and Gamma are not homogeneous.

Reply:
The three parameters do not have the same physical units. Anyway, the parameter R has the strongest effect in this case. The values in the interval seem to be slightly different depending on the machine where the script is evaluated. This why I remove the numerical values.

Reply:
I still disagree with the comment. I don't think it makes sense to compare the sizes of the confidence intervals in general. It could make sense to compare R and C because they are commensurate, but not R and gamma, or C and gamma.

Reply:
To be precise, the thing I disagree with is the idea that R has the smallest confidence interval.

Reply:
The length of the intervals is somewhat hidden by the order of magnitude of the parameters. With a little change in the script, I could print this 95% C.I.:
- R in [730.3, 772.1]
- C in [758.2, 3661.0]
- Gamma in [-1450.7629, 1452.3081]

Let us compute the relative confidence interval for each marginal:
```
>>> [(upperBound[i] - lowerBound[i]) / thetaMAP[i] for i in range(thetaMAP.getDimension())]
[0.05, 1.31, 3757.54]  # (Rounded to 2 digits for readability)
```
Hence, 
- the length of the interval for R represents 5% of the optimum value, 
- the length of the interval for C represents 131% of its value
- and the ratio is more than 3000 for gamma. 

In other words, consider the case where we do not calibrate all the parameters but only a subset of it:
- we could as well calibrate this model using R alone,
- calibrating R and C would improve a little over calibrating only R,
- calibrating R, C and gamma is not necessary.

This is a consequence of the fact that the model is actually much more sensitive to R. Had we done sensitivity analysis before calibrating the model, we would have reduced the dimension to this single parameter. To convince yourself of this, please perform a sensitivity analysis of this model using Sobol' indices. To do this, we can use the same distribution as the one used in the Gaussian calibration. Notice that there is no input distribution in classical least squares. We may use a flat Gaussian prior (as is implicitly done), but this might raise numerical issues for sensitivity analysis (I did not try that).

Do you agree with this analysis?

Reply:
Thank you for the detailed answer! I agree with the new formulation, which removes the ambiguity.